storage/txcache/eviction.go

```diff
@@ -123,18 +96,36 @@ func (cache *TxCache) evictHighNonceTransactions() (uint32, uint32) {
 		}
 	})
 
+	if len(txsToEvict) == 0 {
```

Comment:
this condition could be deleted, as it may be resolved in RemoveTxsBuld and RemoveSendersBulk. If the condition is still wanted I think it should be len(txsToEvict) == 0 && len(sendersToEvict)

Reply:
I've removed the condition, the `range over` will be no-ops, which is fine.